src/abstractitensornetwork.jl

```diff
@@ -331,6 +331,14 @@ end
 adjoint(tn::Union{IndsNetwork,AbstractITensorNetwork}) = prime(tn)
 
 dag(tn::AbstractITensorNetwork) = map_vertex_data(dag, tn)
+#Faster version!
+function fast_dag(tn::AbstractITensorNetwork)
```

Comment:
Could just call this `dag` and keep the other one commented out in the code as a reminder to make a faster version/mode for `map_vertex_data`.